ros_ign_gazebo/scripts/ign_gazebo

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# TODO This approach has way too many intermediate steps:
```

Comment:
I think for ROS 2 it's possible to do something _a bit_ more direct, like this:

https://github.com/ignitionrobotics/ros_ign/blob/82354e4923c5bdfbe239b60654fc765567b74a9a/ros_ign_gazebo_demos/launch/ign_gazebo.launch.py#L27-L37

Reply:
Did you give any thought to this? Is there a downside to calling `ign gazebo` directly from the launch file? I think this bash script adds an unnecessary intermediate step.